build.rs

```diff
@@ -107,6 +107,11 @@ fn make_source() {
     base_config.include(&libusb_source);
     base_config.include(libusb_source.join("libusb"));
 
+    // Include line numbers for debugging libusb C code when performing debug builds
+    if env::var("PROFILE").unwrap_or(String::new()) == "debug" {
+        base_config.flag("-g");
```

Comment:
It's flag only valid for gcc and clang

https://ci.appveyor.com/project/a1ien/libusb1-sys/builds/35118678/job/tu7isqr4ax7ff3an#L62

Reply:
Thanks, didn't think of that. I'll fix it up and re-submit!